src/main/java/com/conveyal/r5/analyst/TravelTimeSurfaceComputer.java

```diff
@@ -0,0 +1,179 @@
+package com.conveyal.r5.analyst;
+
+import com.conveyal.r5.analyst.cluster.AccessGridWriter;
+import com.conveyal.r5.analyst.cluster.TravelTimeSurfaceRequest;
+import com.conveyal.r5.analyst.error.TaskError;
+import com.conveyal.r5.api.util.LegMode;
+import com.conveyal.r5.common.JsonUtilities;
+import com.conveyal.r5.profile.FastRaptorWorker;
+import com.conveyal.r5.profile.PerTargetPropagater;
+import com.conveyal.r5.profile.RaptorWorker;
+import com.conveyal.r5.profile.StreetMode;
+import com.conveyal.r5.publish.StaticComputer;
+import com.conveyal.r5.streets.LinkedPointSet;
+import com.conveyal.r5.streets.StreetRouter;
+import com.conveyal.r5.transit.TransportNetwork;
+import gnu.trove.iterator.TIntIntIterator;
+import gnu.trove.map.TIntIntMap;
+import org.apache.commons.math3.util.FastMath;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.IntStream;
+
+/**
+ * This computes a travel time surface and returns it in access grid format, with one travel time for each minute/MC draw.
```

Comment:
It looks like this Javadoc is incorrect - this class returns a grid with the third dimension representing percentiles of travel time at each destination, in increasing order.